src/nidcpower/metadata/functions_addon.py

```diff
@@ -53,8 +53,6 @@
     'ConfigureDigitalEdgeStartTrigger':             { 'parameters': { 2: { 'enum': 'DigitalEdge',                 }, }, },
     'SendSoftwareEdgeTrigger':                      { 'parameters': { 1: { 'enum': 'SendSoftwareEdgeTriggerType', }, }, },
     'WaitForEvent':                                 { 'parameters': { 2: { 'enum': 'Event',                       }, }, },
-    'Measure':                                      { 'parameters': { 2: { 'enum': 'MeasurementTypes',            }, }, },
-    'QueryOutputState':                             { 'parameters': { 2: { 'enum': 'OutputStates',                }, }, },
     'ExportSignal':                                 { 'parameters': { 1: { 'enum': 'Signals',                     }, }, },
```

Comment:
Removed MeasurementTypes adn QueryOutput state from this PR since there exist another PR for the same